pkg/schemadsl/parser/tests/caveatstype.zed.expected

```diff
@@ -35,7 +35,7 @@ NodeTypeFile
                       caveat-name = somecaveat
                       end-rune = 67
                       input-source = caveats type test
-                      start-rune = 53
+                      start-rune = 58
```

Comment:
This is kind of surprising - is this because `with ` was previously a part of the caveat definition?

Reply:
Correct